polaris-service/src/test/java/org/apache/polaris/service/PolarisApplicationIntegrationTest.java

```diff
@@ -115,7 +123,11 @@ public class PolarisApplicationIntegrationTest {
               "server.applicationConnectors[0].port",
               "0"), // Bind to random port to support parallelism
           ConfigOverride.config(
-              "server.adminConnectors[0].port", "0")); // Bind to random port to support parallelism
+              "server.adminConnectors[0].port", "0"), // Bind to random port to support parallelism
+          ConfigOverride.config("logging.appenders[1].type", "file"),
+          ConfigOverride.config("logging.appenders[1].currentLogFilename", CURRENT_LOG),
+          ConfigOverride.config("logging.appenders[1].archivedLogFilenamePattern", ARCHIVED_LOG),
+          ConfigOverride.config("logging.appenders[1].maxFileSize", "2000000"));
```

Comment:
Just curious, is 2M big enough? Another option is to check archived log as well.

Reply:
maxFileSize is not required. I removed it.